src/usePopover.ts

```diff
@@ -67,6 +70,11 @@ export const usePopover = ({
 
       const isExhausted = positionIndex === positions.length;
       const position = isExhausted ? positions[0] : positions[positionIndex];
+
+      if (position === 'custom') {
+        throw new Error('Custom position not supported without contentLocation being provided.');
```

Comment:
Throw error when position is custom and contentLocation prop is not provided. This requirement is specified in the documentation but not enforced.